examples/widgets/drag_and_drop.enaml

```diff
@@ -97,13 +98,14 @@ enamldef Main(Window):
         pass
     Container:
         constraints = [
-            hbox(vbox(lbl1, lbl2, spacer), target),
+            hbox(vbox(lbl1, lbl2), target),
         ]
         DragLabel: lbl1:
             text = 'Drag Me 1'
-            data = 'small\ndata'
+            data = b'small\ndata'
```

Comment:
Not sure what you mean by stay literal... if it is that it is translated in two characters then I think the code for b-string may be wrong.